### PR TITLE
Deploy a private npm registry in 5 minutes

### PR DESCRIPTION
**Pull request by fengmk2**
see https://github.com/cnpm/cnpmjs.org/wiki/Deploy-a-private-npm-registry-in-5-minutes

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+ *   fengmk2 <m@fengmk2.com> (http://fengmk2.com)
+ */
+
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+require('gnode');
```
Comment (fengmk2) [inline bin/cli.js]: 改用 gnode 了

Comment (fengmk2) [inline bin/cli.js]: 等 https://github.com/mapbox/node-sqlite3/pull/399 合并了，就直接推荐使用 iojs

**Comment by dead-horse** [comment]
:+1:

**Comment by fengmk2** [comment]
我发个 rc13

**Pull request merged**